app/services/actions/foreman/puppet_class/import.rb

```diff
@@ -0,0 +1,30 @@
+module Actions
+  module Foreman
+    module PuppetClass
+      class Import < Actions::EntryAction
+        def resource_locks
+          :import_puppetclasses
+        end
+
+        def run
+          output[:changed] = input[:changed]
```

Comment:
the reason was (sorry I could not remember yesterday) so that other tasks that might trigger this task could get the data and work with them further, not sure if this is needed or there's a better way